client/swagger.yaml

```diff
@@ -195,6 +195,32 @@ paths:
           schema:
             $ref: "#/definitions/Error"
 
+  /cpu-config:
```

Comment:
this is just updating the fc client apis, we still need to add support for this feature in the go-sdk. Can we add a issue to track the compatibility work?

Reply:
Created two issues #505 and #504 to track unimplemented features introduced from the upgrade.